js/hang-ui/src/Components/stats/context.ts

```diff
@@ -0,0 +1,16 @@
+import { createContext, useContext } from "solid-js";
+import type { HandlerProps } from "./types";
+
+/**
+ * Context providing stream sources to child components
+ */
+export const StatsContext = createContext<HandlerProps>({});
+
+/**
+ * Hook to access stream sources from context
+ * @returns Stream properties (audio and video)
+ */
+export const useMetrics = () => {
+	const context = useContext(StatsContext);
```

Comment:
for the other wrappers, we `throw` if context is undefined. Do you think we need to do that here?